pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
```

Comment:
I don't feel strongly about it, but I thought it clarified that this was a use-case.